src/site/content/en/blog/wake-lock/index.md

```diff
@@ -148,7 +148,7 @@ let wakeLock = null;
 // Function that attempts to request a screen wake lock.
 const requestWakeLock = async () => {
   try {
-    wakeLock = await navigator.wakeLock.request('screen');
+    wakeLock = await navigator.wakeLock.request();
```

Comment:
Wouldn't you want to future proof code so that if other values are added and `'screeen'` is not the default, the code will still work?

Reply:
The W3C Working Group (that I am part) of has decided that `"screen"` is the default and will be.